_data/examples.json

```diff
@@ -159,6 +159,10 @@
       "title": "Simple Bar Chart with Labels",
       "style": "background-size: auto 70%; background-position-y: center;"
     },
+    {
+      "name": "layer_bar_annotations",
+      "title": "Bar chart with highlighting annotation overlay"
```

Comment:
Be a bit more specific: e.g., "Bar chart that highlights a value beyond a threshold"?

Also add a `description` field trying to describe the example -- this will appear in your example page like in https://vega.github.io/vega-lite/examples/bar.html. 

(You can use markdown for the `description` field.)